src/modelinstance.cpp

```diff
@@ -695,14 +695,7 @@ Status ModelInstance::loadOVCompiledModel(const ModelConfig& config) {
     }
 
     uint32_t numberOfStreams = 0;
-    try {
-        numberOfStreams = getNumOfStreams();
-        SPDLOG_LOGGER_INFO(modelmanager_logger, "Number of OpenVINO streams: {}", numberOfStreams);
-    } catch (ov::Exception& e) {
-        SPDLOG_LOGGER_WARN(modelmanager_logger, "Unable to get information about number of streams with error: {}; model: {}; version: {}; device: {}", e.what(), getName(), getVersion(), config.getTargetDevice());
-    } catch (...) {
-        SPDLOG_LOGGER_WARN(modelmanager_logger, "Unable to get information about number of streams; model: {}; version: {}; device: {}", getName(), getVersion(), config.getTargetDevice());
-    }
+    getNumOfStreams(numberOfStreams);
```

Comment:
Wouldnt it be better to change return type to `uint32_t` and return 0 in case of errors? I think it would be more clean